install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -42,7 +42,7 @@ spec:
           - containerPort: 15010
           readinessProbe:
             httpGet:
-              path: /v1/registration
+              path: /debug/edsz
```

Comment:
roughly equivalent new api

Reply:
we probably want to use something else in the future, if we have 10k endpoints, we don't necessarily want to marshal them all to json for no reason.